File: src/main/java/dev/slne/discord/exception/ticket/member/TicketMemberException.java
package dev.slne.discord.exception.ticket.member;

import dev.slne.discord.exception.DiscordException;
import java.io.Serial;

public abstract class TicketMemberException extends DiscordException {

  @Serial
  private static final long serialVersionUID = 8277057907153655874L;

  public TicketMemberException(String message) {
    super(message);
  }

  public TicketMemberException(String message, Throwable cause) {
    super(message, cause);
  }

  public TicketMemberException(DiscordException exception) {
    super(exception);
  }
}

File: src/main/kotlin/dev/slne/discord/exception/ticket/member/TicketMemberException.kt
package dev.slne.discord.exception.ticket.member

import dev.slne.discord.exception.DiscordException
import java.io.Serial

abstract class TicketMemberException : DiscordException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    constructor(exception: DiscordException) : super(exception)

    companion object {
        @Serial
        private const val serialVersionUID = 8277057907153655874L
    }
}
